tests/test_generate_model_schemas.py

```diff
@@ -62,7 +62,6 @@ def test_generate_schema():
         "required": [
             "id",
             "framework_id",
-            "is_a_copy",
```

Comment:
Why is this unexpected change safe?

Reply:
@bjgill I have done some more research into this, with assistance from @alex9smith please see the extended commit message in 06c65a8 for more details. See also definition of `is_a_copy` from https://github.com/alphagov/digitalmarketplace-api/blob/e781077b52797a963a2a0663bf8e2a5463813cb5/migrations/versions/900_add_brief_is_a_copy.py#L18

Reply:
That makes sense - thanks for the explanation